### PR TITLE
[DOC] Update monitoring doc

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:
Updates the monitoring Tempo documentation with instructions on how to use the mixins. 

Remaining
- [x] Add screenshots
- [x] Proofread
- [ ] SME review

**Which issue(s) this PR fixes**:
Fixes #https://github.com/grafana/tempo-squad/issues/344

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/operations/monitor/set-up-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
+---
+title: Set up monitoringfor Tempo
```
Comment (knylander-grafana) [inline docs/sources/tempo/operations/monitor/set-up-monitoring.md:2]:
```suggestion
title: Set up monitoring for Tempo
```

File: docs/sources/tempo/operations/monitor/set-up-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+description: Set up monitoring for Tempo
+weight: 20
+---
+
+# Set up monitoring for Tempo
+
+You can set up monitoring for Tempo using an existing or new cluster.
+If you don't have a cluster available, you can use the linked documentation to set up the Tempo, Mimir, and Grafana using Helm or you can use Grafana Cloud.
+
+To set up monitoring, you will:
```
Comment (knylander-grafana) [inline docs/sources/tempo/operations/monitor/set-up-monitoring.md:13]:
```suggestion
To set up monitoring, you need to:
```

**Review by zalegrala — APPROVED** [review]
Nice, this looks good to me.  Good work.

**Comment by knylander-grafana** [comment]
> Nice, this looks good to me. Good work.

Thank you, Zach!

**Comment by knylander-grafana** [comment]
@hedss  approved in a Slac message-- said to merge.

**Pull request merged**